core/src/main/scala/libretto/CoreLib.scala

```diff
@@ -1591,6 +1589,8 @@ class CoreLib[DSL <: CoreDSL](val dsl: DSL) { lib =>
 
   given Bifunctor[|&|] = |&|.bifunctor
 
+  // TODO: Converting to `extension` would require rename of methods as they clash with the DSL,
+  //  or moving of this definitions to a place where the name clash could be avoided.
   implicit class LinearFunctionOps[A, B](self: A -⚬ B) {
```

Comment:
The comments says it all. These implicit classes need to move I guess…

But didn't touch as I don't know where to put things, and have a poor overview until now anyway.

Reply:
Let's move to a separate issue.